.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+_When issuing a bug report please include as many of the below details as possible. If you find a Closed issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one._
+
+**Describe the bug**
+
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behaviour:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Reproduces how often:**
+
+What percentage of the time does it reproduce, if you know?
+
+
+**Expected behaviour**
```

Comment:
Need to sort whitespacing on these if they're designed to be different sections. (same below)

Reply:
done